lib/matplotlib/ticker.py

```diff
@@ -957,22 +957,31 @@ def set_locs(self, locs=None):
             return
 
         b = self._base
+        def log_b(x): return math.log(x) / math.log(b)
```

Comment:
`math.log(x, b)` and therefore maybe no need for a function?

Reply:
I never realized log() takes two arguments, but sure, yes.